tests/test_workon.py

```diff
@@ -18,7 +18,7 @@ def test_detect_shell():
         except KeyError:
             pass
         if sys.platform == 'win32':
-            assert _detect_shell() in ['python.exe']
+            assert _detect_shell() in ['cmd.exe']
```

Comment:
This fails if the tests are run from a shell other than cmd (for example, I use powerrshell as my standard shell). I don't think this is the end of the world (CI uses cmd, which is the most important thing) but maybe a comment in the test to help people who hit a failure would be useful.

FWIW, the failure looks like this:

```
______________________________________________________________________________ test_detect_shell ______________________________________________________________________________
                                                                                                                                                                               
    def test_detect_shell():                                                                                                                                                   
        with temp_environ():                                                                                                                                                   
            try:                                                                                                                                                               
                del os.environ['SHELL']                                                                                                                                        
            except KeyError:                                                                                                                                                   
                pass                                                                                                                                                           
            if sys.platform == 'win32':                                                                                                                                        
>               assert _detect_shell() in ['cmd.exe']                                                                                                                          
E               AssertionError: assert 'powershell.exe' in ['cmd.exe']                                                                                                         
E                +  where 'powershell.exe' = _detect_shell()                                                                                                                   
                                                                                                                                                                               
tests\test_workon.py:21: AssertionError                                                                                                                                        
============================================================== 1 failed, 40 passed, 22 skipped in 790.38 seconds ==============================================================
ERROR: InvocationError for command 'C:\Work\Projects\pew\.tox\py37\Scripts\py.test.EXE' -rw (exited with code 1)                                                               
```

Reply:
I also added a marker for people to disable the test.